### ISSUE TITLE
wrong results with 2 terminals open at once

### ISSUE THREAD

**Issue by isomorphisms** [hubsmoke/bro#69]
`terminal 1 ~ $ bro join`

`terminal 2 ~ $ bro lam`

`terminal 1 ~ $ bro thanks`

> ` that index (1) is not available for 'lam'`

**Comment by hubsmoke** [comment]
This is true, the CLI only caches the last command entered. So when you have multiple terminals open it will screw it up. I suppose one way to fix this would be to assign each session it's own UUID and then cache it by UUID and then maybe expire the UUID after a certain amount of time so the cache file doesn't grow too big
